content/en/docs/test.md

```diff
@@ -369,7 +369,9 @@ Notes catch the reader's attention without a sense of urgency.
 
 You can have multiple paragraphs and block-level elements inside an admonition.
 
-| Or | a | table |
+| Or | | Even |
+|:---|:---:|-----:|
+|   A    | |   Table    |
```

Comment:
From the website, we cannot tell the different about align. And the empty column is invisible for human eye.

I think that's unnecessary to add the complex.

Reply:
I thought that it will be more readable to users... I will remove it then